internal/api/v1alpha1/authproxyworkload_types.go

```diff
@@ -58,27 +58,40 @@ const (
 	ReasonUpToDate = "UpToDate"
 )
 
-// AuthProxyWorkloadSpec defines the desired state of AuthProxyWorkload
```

Comment:
These are documentation changes intended to improve published api documentation.